components/aws/sagemaker/TrainingJob/samples/mnist-kmeans-classification/README.md

```diff
@@ -0,0 +1,82 @@
+# MNIST Classification with KMeans
+
+This sample aims to showcase the usage of v1 components and the new v2 training job component together in a Kubeflow pipeline workflow. The `mnist-classification-pipeline.py` sample runs a pipeline to train a classficiation model using Kmeans with MNIST dataset on SageMaker. This example was taken from an existing [SageMaker example](https://github.com/awslabs/amazon-sagemaker-examples/blob/master/sagemaker-python-sdk/1P_kmeans_highlevel/kmeans_mnist.ipynb) and modified to work with the Amazon SageMaker Components for Kubeflow Pipelines.
+
+## Prerequisites 
+
+1. Make sure you meet all the configuration prerequisites *including the optional step* for using the v2 components from sagemaker/TrainingJob README (https://github.com/kubeflow/pipelines/tree/master/components/aws/sagemaker/TrainingJob). 
+2. The optional step in the guide (https://github.com/kubeflow/pipelines/tree/master/components/aws/sagemaker/TrainingJob) is required for this sample to run. Here are the commands in case you missed it:
+    1. Grant SageMaker access to the service account used by kubeflow pipeline pods. Export your cluster name and cluster region
+    2. export CLUSTER_NAME=
+       export CLUSTER_REGION=
+    3. eksctl create iamserviceaccount --name ${KUBEFLOW_PIPELINE_POD_SERVICE_ACCOUNT} --namespace ${PROFILE_NAMESPACE} --cluster ${CLUSTER_NAME} --region ${CLUSTER_REGION} --attach-policy-arn arn:aws:iam::aws:policy/AmazonSageMakerFullAccess --override-existing-serviceaccounts --approve
```

Comment:
Weird font here for namespace, is it a github issue? https://github.com/kubeflow/pipelines/blob/73d45eeb7136bb23f9d7e103f50a226f111c3143/components/aws/sagemaker/TrainingJob/samples/mnist-kmeans-classification/README.md

Reply:
fixed